test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -6266,7 +6293,10 @@ def submit(self, obj, script=None, extend=None, submit_dir=None):
             return None
 
         if submit_dir is None:
-            submit_dir = tempfile.gettempdir()
+            if self.platform == 'cray' or self.platform == 'craysim':
+                submit_dir = pwd.getpwnam(obj.username)[5]
```

Comment:
I think we should always submit job from user's home directory if submit_dir is None. What say?